### ISSUE TITLE
Hunter: Lone Wolf / Tame Beast and Hunter's Mark

### ISSUE THREAD

**Issue by orjanbp** [WC5E/Warcraft-5e-Conversion#625]
**Problem**
A very good solution we have right now for solving Lone Wolf vs Tame Beast is to make the damage-dealing component of Hunter's Mark the counterpoint to Tame Beast. However, there's a very strong feeling against losing out on Hunter's Mark over not picking Lone Wolf. 

**Discussed solution**
* Keep Hunter's Mark as a core feature focused on tracking (with it, are we looking at a small change or are we looking at re-imagining it as a very different feature from how it is now?). 

* Take the damage-dealing part of Hunter's Mark, as it is, and apply it solely to Lone Wolf. 

* In other words, Hunter's Mark is always there. Every Hunter has it and can use it to track. The only thing that differs is the extra damage.

**Food for thought**
* Would people expect Hunter's Mark to have a definite combat role for a hunter with a beast? 

Add more underneath!

**Comment by orjanbp** [comment]
Also

* For the way these new beasts work, we need to figure out the mechanics / economics of replacing it in the future. The first one (at 1st level) would obviously be free, since it comes with the character creation.

**Comment by Frogvall** [comment]
Rough initial draft:
https://www.gmbinder.com/share/-M7DISsOURwET5f14UID

**Comment by Tyloris** [comment]
A simplified version of this was added in the latest hunter revision.
